showcase/types/global.d.ts

```diff
@@ -1,35 +0,0 @@
-/**
- * Copyright (c) HashiCorp, Inc.
- * SPDX-License-Identifier: MPL-2.0
- */
-
-import '@glint/environment-ember-loose';
```

Comment:
You might have deleted a little too much here we need to keep this two lines for glint to work